api/proto/config/devmodel.proto

```diff
@@ -117,7 +117,14 @@ message PhysicalIO {
       // Example Eth0, Eth1, Wifi0, ComA, ComB
       string phylabel           = 2;
 
-      // the hardware bus address
+      // The hardware bus address. The key to this map can be of the following
+      // (case-insensitive) values:
+      // "pcishort": the address is a PCI id of the form 02:00.0
+      // "pcilong": the address is a PCI id of the form 0000:02:00.0
```

Comment:
What's the reason of allowing both short and long PCI addresses? Why not standardize on just the long ones? Or is the idea here to assume that short always means the PCI domain is 0000? If so it needs to be documented.

Reply:
You're right, we can just use the long format with the domain.
I'll update the proto file.